clang/test/dpct/thrust-for-h2o4gpu.cu

```diff
@@ -1,6 +1,6 @@
 // UNSUPPORTED: cuda-8.0
 // UNSUPPORTED: v8.0
-// RUN: dpct --format-range=none -out-root %T/thrust-for-h2o4gpu %s --cuda-include-path="%cuda-path/include" -- -std=c++14 -x cuda --cuda-host-only -fno-delayed-template-parsing
+// RUN: dpct --format-range=none -out-root %T/thrust-for-h2o4gpu %s --cuda-include-path="%cuda-path/include" -- -std=c++14 -x cuda --cuda-host-only -fno-delayed-template-parsing -ferror-limit=50
```

Comment:
`-ferror-limit=50` is to allow more parsing errors than default 20, otherwise dpct will stop migration due to two many errors if CUDA_PATH is 11.2~11.7

For dpct/thrust-for-h2o4gpu:
16 parsing errors are reported with CUDA_PATH set to 11.0 and 11.1.
22 parsing errors are reported with CUDA_PATH set to 11.2+.

If "-ferror-limit" is not passed, the error limit is 20 by default.

The other 3 LIT tests have the same issue.